awx/ui_next/src/components/Breadcrumbs/Breadcrumbs.jsx

```diff
@@ -7,26 +7,24 @@ import {
   BreadcrumbItem,
   BreadcrumbHeading,
 } from '@patternfly/react-core';
-import { Link, Route, withRouter } from 'react-router-dom';
+import { Link, Route, withRouter, useRouteMatch } from 'react-router-dom';
+
 import styled from 'styled-components';
 
 const PageSection = styled(PFPageSection)`
   padding-top: 10px;
   padding-bottom: 10px;
 `;
 
-const Breadcrumbs = ({ breadcrumbConfig }) => {
+const Breadcrumbs = ({ breadcrumbConfig, match }) => {
```

Comment:
Also passing match...It'd be better to get what we need in Crumb using hooks I think